src/app/shared/material/material.module.ts

```diff
@@ -16,6 +16,9 @@ import { MatIconModule } from '@angular/material/icon';
 import { MatTooltipModule } from '@angular/material/tooltip';
 import { MatCardModule } from '@angular/material/card';
 import { MatDividerModule } from '@angular/material/divider';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatSelectModule } from '@angular/material/select';
+import { MatInputModule } from '@angular/material/input';
 
 
 @NgModule({
```

Comment:
You have different amount of empty lines after imports (in some files - 1 line, on other - 2). Please make sure you have same amount of empty lines after imports in every ts file

Reply:
fixed